lib/ReactViews/FeatureInfo/FeatureInfoSection.jsx

```diff
@@ -111,6 +111,7 @@ const FeatureInfoSection = createReactClass({
 
     descriptionFromTemplate() {
         const template = this.props.template;
+        const safeGuardRegex = new RegExp("=(\\s*{{.+?}})","g");
```

Comment:
Please use single quotes for strings within TerriaJS (yes, it's different from Magda).  Also you might want to use `/whatever/g` syntax rather than an explicit RegExp constructor.